src/azure-cli/azure/cli/command_modules/monitor/commands.py

```diff
@@ -277,15 +347,39 @@ def load_command_table(self, _):
         g.show_command('show', 'get')
         g.command('delete', 'delete')
         g.custom_command('list', 'list_log_analytics_workspace')
-        g.command('get-schema', 'get_schema')
-        g.command('list-usages', 'list_usages')
-        g.command('list-management-groups', 'list_management_groups')
-        g.command('get-shared-keys', 'get_shared_keys')
-
-    with self.command_group('monitor log-analytics workspace pack', log_analytics_workspace_sdk, custom_command_type=log_analytics_workspace_custom) as g:
-        g.command('list', 'list_intelligence_packs')
-        g.command('enable', 'enable_intelligence_pack')
-        g.command('disable', 'disable_intelligence_pack')
+        g.command('get-schema', 'get', command_type=log_analytics_workspace_schema_sdk)
+        g.command('list-usages', 'list', command_type=log_analytics_workspace_usage_sdk)
+        g.command('list-management-groups', 'list', command_type=log_analytics_workspace_management_groups_sdk)
+        g.command('get-shared-keys', 'get_shared_keys', command_type=log_analytics_workspace_shared_keys_sdk)
+
+    with self.command_group('monitor log-analytics workspace pack', log_analytics_workspace_intelligence_packs_sdk) as g:
+        g.command('list', 'list')
+        g.command('enable', 'enable')
+        g.command('disable', 'disable')
+
+    with self.command_group('monitor log-analytics workspace linked-service', log_analytics_workspace_linked_service_sdk, custom_command_type=log_analytics_workspace_linked_service_custom) as g:
+        g.custom_command('create', 'create_log_analytics_workspace_linked_service', supports_no_wait=True)
+        g.generic_update_command('update', custom_func_name='update_log_analytics_workspace_linked_service', supports_no_wait=True)
+        g.show_command('show', 'get')
+        g.command('list', 'list_by_workspace')
+        g.command('delete', 'delete', confirmation=True, supports_no_wait=True)
+        g.wait_command('wait')
+
+    with self.command_group('monitor log-analytics cluster', log_analytics_cluster_sdk, custom_command_type=log_analytics_cluster_custom, is_preview=True) as g:
+        g.custom_command('create', 'create_log_analytics_cluster', validator=make_cluster_creation_as_no_wait_by_default)
+        g.custom_command('update', 'update_log_analytics_cluster')
+        g.show_command('show', 'get')
+        g.command('delete', 'delete', confirmation=True, supports_no_wait=True)
+        g.custom_command('list', 'list_log_analytics_clusters')
+        g.wait_command('wait')
+
+    with self.command_group('monitor log-analytics workspace linked-storage', log_analytics_linked_storage_sdk, custom_command_type=log_analytics_linked_storage_custom, is_preview=True) as g:
```

Comment:
As "with self.command_group('monitor', metric_alert_sdk, custom_command_type=monitor_general_custom, is_preview=True) " already defined as `preview`, can we just remove "is_preview=True" here?